content/docs/coldchain/monitoring.md

```diff
@@ -0,0 +1,134 @@
++++
+title = "Monitoring"
+description = "Temperature monitoring and reporting features"
+date = 2023-11-08T15:20:00+00:00
+updated = 2023-11-08T15:20:00+00:00
+draft = false
+weight = 1
+sort_by = "weight"
+template = "docs/page.html"
+
+[extra]
+toc = true
+top = false
++++
+
+The Monitoring section displays cold chain monitoring data in three different views:
+1. A chart view which shows a summary of data from all sensors
+2. A list of all recorded breaches
+3. A list view of all temperature logs
+
+### Viewing Monitoring data
+
+Choose `Cold chain` > `Monitoring` in the navigation panel.
+
+![goto monitoring](/docs/coldchain/images/goto_monitoring.png)
+
+This page contains three tabs, displaying different views of temperature monitoring data. The tab which will be displayed by default is the `Chart` tab.
+
+### Chart
+
+![Chart view](/docs/coldchain/images/chart_view.png)
+
+The chart tab displays a chart showing all configured sensors. You can filter the data displayed by:
+
+- Sensor name
+- Location
+- Date/Time Range
+- Breach type
+
+To add a filter to the page, choose the required filter from the drop down. 
+
+![Filter](/docs/coldchain/images/log_filter.png)
+
+If the sensor has recorded a breach - an icon will be shown on the chart. Clicking on the icon displays information about the breach, with a button which takes you to the list of all breaches:
+
+![Chart Breach](/docs/coldchain/images/chart_breach.png)
+
+To close the popup, click the red x in the top right corner.
+
+Also shown on the chart are red (top) and blue (bottom) shaded areas. These show the standard temperature breach threshold values of 2°C and 8°C. Note that your environment may be using different threshold values, so you may have breaches which are outside of these ranges!
+
+### Breaches
+
+The breaches tab displays a list of all recorded breaches. 
+
+![Breaches](/docs/coldchain/images/breaches_list.png)
+
+The list of breaches is divided into 10 columns:
+
+| Column              | Description                      |
+| :------------------ | :------------------------------- |
+|                     | An icon showing an alert if this breach is unacknowledged        |
+| **Status**          | The breach status can be `Acknowledged` or `Unacknowledged`     |
+| **Sensor name**     | Name of the Sensor               |
+| **Location**        | Current location assigned to the sensor |
+| **CCE**             | Name of the cold chain equipment item which the sensor is associated with   |
+| **Breach start**    | Date and time at which the breach started |
+| **Duration**     | The duration of the breach, if it has ended. If the breach has not ended then `Ongoing` is shown |
```

Comment:
I didn't know it showed Ongoing 🤔 ... I do see the label though in om, but it's not being used 👀

Reply:
It's now in v1.5.0 - coming from [this pr](https://github.com/msupply-foundation/open-msupply/pull/2439/files#diff-39160c843a1e41bf90b0c62a3b1fc4f71338b6f6487e4d4e73ff84383f05f9d6)